src/DatePicker/DatePicker.test.js

```diff
@@ -62,33 +71,57 @@ describe('<DatePicker />', () => {
         //open date picker calendar
         expect(wrapper.state('hidden')).toBeTruthy();
 
+        // click to show calendar
         wrapper.find('input[type="text"]').simulate('click', { type: 'input' });
 
+        // check to make sure calendar is shown
         expect(wrapper.state('hidden')).toBeFalsy();
 
         wrapper.instance().componentWillMount();
 
+        // click on body element
         let event = new MouseEvent('mousedown', {
             target: document.querySelector('body')
         });
         document.dispatchEvent(event);
 
+        // check to make sure calendar is hidden
         expect(wrapper.state('hidden')).toBeTruthy();
 
         // show date picker, select date range then close
         wrapper.find('input[type="text"]').simulate('click', { type: '' });
 
+        // check to make sure calendar is shown
         expect(wrapper.state('hidden')).toBeFalsy();
+    });
 
+    test('start date and end date range', () => {
         wrapper = mount(rangeDatePicker);
+        // set dates
         let startRangeDate = new Date();
         let endRangeDate = new Date();
         endRangeDate.setDate(endRangeDate.getDate() + 3);
 
         let arrDates = [startRangeDate, endRangeDate];
         wrapper.instance().updateDate(arrDates);
 
+        expect(wrapper.state('arrSelectedDates').length).toEqual(2);
+
+        // click on body element
+        let event = new MouseEvent('mousedown', {
+            target: document.querySelector('body')
+        });
         document.dispatchEvent(event);
```

Comment:
I don't understand why the added `mousedown` event, but then no additional assertions.  I'm probably just missing something.

Reply:
removed MouseEvent code since it was not necessary

Reply:
sorry, missed that mouseDown event. Just added an assertion to check that calendar is hidden.